README.md

```diff
@@ -1,115 +1,38 @@
 
-&lt;REPLACE ME DESCRIPTION START&gt;
+- [Salesforce DX Setup Guide](https://sfdc.co/sfdx_setup_guide)
+- [Salesforce DX Developer Guide](https://sfdc.co/sfdx_dev_guide)
+- [Salesforce CLI Command Reference](https://sfdc.co/sfdx_cli_reference)
+- [Salesforce Developer Tools for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=salesforce.salesforcedx-vscode)
 
-This repository provides a template for creating a plugin for the Salesforce CLI. To convert this template to a working plugin:
+## Installing
 
-1. Clone this repo
-2. Delete the .git folder
-3. Replace filler values
-   a) Every instance of `<REPLACE_ME>` can be directly substitued for the name of the new plugin. However beware, things like github paths are for the salesforcecli Github organization
-   b) Search for case-matching `REPLACE` to find other filler values, such as for the plugin description
-4. Use `git init` to set up the desired git information
-5. Follow the getting started steps below until the `sfdx hello:org` commmand is functioning
-6. In order to prevent build failures on the intial build, you will need to do the following:
-   1. `npm publish`
-   2. `git tag v1.0.0 ; git push origin v1.0.0`
+Download and install the [Salesforce DX CLI](https://developer.salesforce.com/tools/sfdxcli).
 
-&lt;REPLACE ME DESCRIPTION END&gt;
+By default, the Salesforce DX plugin is bundled with the cli:
 
-### Everything past here is only a suggestion as to what should be in your specific plugin's descsription.
-
-## Getting Started
-
-To use, install the [Salesforce CLI](https://developer.salesforce.com/tools/sfdxcli) and run the following commands.
-
-```
-Verify the CLI is installed
-  $ sfdx (-v | --version)
-Install the <REPLACE_ME> plugin
-  $ sfdx plugins:install <REPLACE_ME>
-To run a command
-  $ sfdx [command]
-```
-
-To build the plugin locally, make sure to have yarn installed and run the following commands:
-
-```
-Clone the repository
-  $ git clone git@github.com:salesforcecli/plugin-<REPLACE_ME>
-Install the dependencies and compile
-  $ yarn install
-  $ yarn prepack
-Link your plugin to the sfdx cli
-  $ sfdx plugins:link .
-To verify
-  $ sfdx plugins
+```bash
+sfdx plugins --core
 ```
 
-## Debugging your plugin
-
-We recommend using the Visual Studio Code (VS Code) IDE for your plugin development. Included in the `.vscode` directory of this plugin is a `launch.json` config file, which allows you to attach a debugger to the node process when running your commands.
+### Install a "tagged" version of Salesforce DX
 
-To debug the `hello:org` command:
+Use this to install a previous version of Salesforce DX or install a version that has not been promoted to the "latest" version.
 
-If you linked your plugin to the sfdx cli, call your command with the `dev-suspend` switch:
+For instances that are on pre-release:
 
-```sh-session
-$ sfdx hello:org -u myOrg@example.com --dev-suspend
+```bash
+sfdx plugins:install salesforcedx@pre-release
```

Comment:
I wonder if we want to make the example about installing latest-rc instead of pre-release since we're most likely moving away from pre-release?  @amphro

Reply:
Ya, let's do that, and even explain the latest-rc schedule a little bit.